CRM/Admin/Page/Job.php

```diff
@@ -136,10 +136,8 @@ public function run() {
 
   /**
    * Browse all jobs.
-   *
-   * @param null $action
```

Comment:
It's almost like foreshadowing that the variable was declared as type null...

Anyway, agreed the parameter isn't used in either function.